src/redux/actions/categories.js

```diff
@@ -0,0 +1,17 @@
+// constants
+export const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
Please ensure that the value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS` according to the project requirement.

Reply:
done